README.md

```diff
@@ -1,2 +1,41 @@
-# Blossom-UI
- 
+<div align="center">
+
+# Blossom UI
+
+"Worry more about functionality, less about styling. Let Blossom UI take care of styling of your project."
+
+![Screenshot (325)](https://user-images.githubusercontent.com/70198636/155023742-31d6bfdd-ccb7-46ee-a6c2-2caec1c5bc30.png)
+
+</div>
+
+---
+
+## Introduction
+
+Start using Blossom UI in your project just by adding below style tag in your HTML file above all the other styling.
+
+```html
+    <link
+        rel="stylesheet"
+        href="https://ui-blossom.netlify.app/main-css/main.css"
+    />
+```
+
+### Blossom UI contains the following components
+
+- Alert
+- Avatar
+- Badge
+- Button
+- Card
+- Grid
+- Image
+- Input
+- Navigation
+- Typography
```

Comment:
Just a suggestion: Can add a link here with your component examples as well

Reply:
Thanks for suggestions